python/cudf/dataframe/column.py

```diff
@@ -17,10 +18,19 @@
 
 from cudf.utils import cudautils, utils, ioutils
 from cudf.dataframe.buffer import Buffer
+from cudf.dataframe import columnops
 from cudf.bindings.cudf_cpp import count_nonzero_mask, column_view_pointer
 from cudf.bindings.concat import _column_concat
 
 
+@cuda.jit
+def boolean_assign_kernel(data, mask, out):
+    idx = cuda.threadIdx.x + cuda.threadIdx.y * cuda.blockDim.x
```

Comment:
Given you're just working with 1d arrays, you can just do `idx = cuda.grid(1)` here for convenience.